plugins/obs-ffmpeg/obs-ffmpeg-srt.h

```diff
@@ -0,0 +1,836 @@
+/*
+ * The following code is a port of FFmpeg/avformat/libsrt.c for obs-studio.
+ * Port by pkv@obsproject.com.
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#pragma once
+#include <obs-module.h>
+#include "obs-ffmpeg-url.h"
+#include <srt/srt.h>
+#include <libavformat/avformat.h>
+#ifdef _WIN32
+#pragma comment(lib, "ws2_32.lib")
+#endif
```

Comment:
Shouldn't this be part of the CMakefile?

Reply:
i've moved it back to cmake where it was initially; can't remember why i moved it here, was 5 months ago ...